wormdatamodel/data/recording.py

```diff
@@ -1,7 +1,9 @@
-import numpy as np
 import os
 import json
 import pickle
+
+import numpy as np
```

Comment:
Moving this line was a minor style change - styling patterns such as `isort` recommend separating core Python libraries from third-party modules from the 'local' module

But if you prefer the first way just let me know; I didn't make this change holistically, but if you'd like that as well the easiest way would just be to setup `pre-commit` on the repo